static/styles/zulip.css

```diff
@@ -2679,3 +2679,7 @@ button.topic_edit_cancel {
     visibility: visible;
     opacity: 1;
 }
+
+.email_copied {
+    color: #69ca37;
+}
```

Comment:
This should go near the `my_email` CSS.  Also, set `cursor: pointer` on `my_email`, so it's obvious to users it's a thing you can click.

Reply:
Ah yeah, i have that fix locally. Also i was planning to add the clone icon on hover so user is not surprised when the message is popped up suddenly saying `Email copied`.